pkg/auth/artifact_verifier_test.go

```diff
@@ -0,0 +1,29 @@
+package auth
+
+import (
+	"os"
+	"path/filepath"
+	"runtime"
+	"testing"
+
+	"github.com/square/p2/pkg/logging"
+	"github.com/square/p2/pkg/uri"
+	"github.com/square/p2/pkg/util"
+)
+
+func TestManifestVerifierAuthorizesValidBuild(t *testing.T) {
```

Comment:
any interest at all in shoring it up with the following tests?

* reject a build that is signed by something, but not in the keyring
* reject a build not signed at all